.github/workflows/housekeeping.yml

```
@@ -0,0 +1,20 @@
+name: Housekeeping
+on:
+  schedule:
+    - cron: "0 0 1 * *"
+jobs:
+  clean-ghcr:
+    name: Delete old container images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete old releases
+        uses: snok/container-retention-policy@v2
+        with:
+          image-names: aimaas/aimaas-api, aimaas/aimaas-ui
```

For consistency with how we do it in `on_tag.yml`?
```suggestion
          image-names: ${GITHUB_REPOSITORY,,}/aimaas-api, ${GITHUB_REPOSITORY,,}/aimaas-ui
```
Although I have to admit that I don't know what the two commas in the curly braces do...

---

> what the two commas in the curly braces do

Bash parameter expansion which converts the variable name to lower case.